camera/src/index.ts

```diff
@@ -1,14 +1,18 @@
 import type { Express, NextFunction, Request, Response } from 'express'
 import express from 'express'
-import mongoose from 'mongoose'
 import { config } from 'dotenv'
 import { jwtManager } from './utils/JWTManager.js'
+import cors from 'cors'
+import process from 'process'
```

Comment:
cors on the camera is needed or it was for the test?

Reply:
Maybe we need that